test/3x/torch/quantization/weight_only/hqq/test_auto_accelerator.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+import pytest
+import torch
```

Comment:
@xin3he   @WeiweiZhang1 @YIYANGCAI @yiliu30  As we support multiple algorithms in INC, it would be beneficial to help users identify the preferred algorithm for their specific scenarios. We could present some of the accuracy data, tuning cost, model supported, etc.

Reply:
Yes, I agree that. We are building an auto testing system and @chensuyue is following that.